dev-libs/paho-mqtt-c/paho-mqtt-c-1.3.12.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit cmake python-any-r1 toolchain-funcs
+
+MY_TEST_UTILS="paho.mqtt.testing"
```

Comment:
Personally, I would consider declaring those as local variables in src_configure.